### PR TITLE
Add Chart.js

### LINKED ISSUE #3514: Chart.JS icon
**Name:Chart JS**
**Website:https://www.chartjs.org/**
**Alexa rank: 12,269**
  

**Official resources for icon and color:https://www.chartjs.org/img/chartjs-logo.svg**

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Chart.js

**Issue:** Resolves #3514
**Alexa rank:** [~13.7k](https://www.alexa.com/siteinfo/chartjs.org)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon based on [this SVG](https://www.chartjs.org/img/chartjs-logo.svg), colour from [website](https://www.chartjs.org/) stylesheet.

Alternative treatment 1:

Alternative treatment 2:

Alternative treatment 3:

Alternative treatment 4:

Alternative treatment 5:

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
Happy with this first treatment of the icon, thanks @PeterShaggyNoble!
Will wait for at least another @simple-icons/maintainers to agree before merging, given you've presented a load of options.

**Review by service-paradis — APPROVED** [review]
I agree with @adamrusted, the 1st treatment LGTM 📈 
 Thank you @PeterShaggyNoble!

**Pull request merged**
